### PR TITLE
Update readme file

### PR DESCRIPTION
**Pull request by man4toman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Participants will get the most from this lesson if they have familiarity with:
 >
 > _For example:_
 >
-> *   Do you want to make changes to your theme yourself?
```
Comment (man4toman) [inline README.md]: First, I change it to 'make', but after a quick check I figured out it must be plural, then I change it to 'makes' again.

Comment (JulieKuehl) [reply README.md]: I believe that the singular form would be best here but I think you changed it back.

```suggestion
> *   Do you want to make changes to your theme yourself?
```

**Comment by JulieKuehl** [comment]
Thank you for your contribution! Your pull request has been merged.

File: README.md (hunk 1 above)
Comment (man4toman) [reply README.md]: OK, then I'll create new PR for it :)

**Pull request merged**